apps/onlyoffice.sh

```diff
@@ -21,31 +18,58 @@ debug_mode
 root_check
 
 # Nextcloud 13 is required.
-lowest_compatible_nc 13
+lowest_compatible_nc 18
 
 # Test RAM size (2GB min) + CPUs (min 2)
 ram_check 2 OnlyOffice
 cpu_check 2 OnlyOffice
 
-# Notification
-msg_box "Before you start, please make sure that port 80+443 is directly forwarded to this machine!"
-
 # Get the latest packages
 apt update -q4 & spinner_loading
 
-# Check if Nextcloud is installed
-print_text_in_color "$ICyan" "Checking if Nextcloud is installed..."
-if ! curl -s https://"${NCDOMAIN//\\/}"/status.php | grep -q 'installed":true'
+# Check if onlyoffice is already installed
+print_text_in_color "$ICyan" "Checking if Onlyoffice is already installed..."
+if version_gt "$CURRENTVERSION" "18.0.1" && ! does_this_docker_exist 'onlyoffice/documentserver'
```

Comment:
`&& ! does_this_docker_exist 'onlyoffice/documentserver'`

This was here to be able to do it the legacy way instead. So probably not needed here, if we get rid of the legacy way.